Makefile

```diff
@@ -93,7 +93,7 @@ publish: build minify
 # https://stackoverflow.com/a/44249374/10045846
 
 yarn.lock: node_modules package.json
-	${YARN} install --check-cache
+	${YARN} install
```

Comment:
See my other comment - I think using `${YARN} install ${YARNFLAGS}` would get us the best of both worlds.